### PR TITLE
Add a login link 

### PR DESCRIPTION
**Pull request by gkwan-ibm**
To make the cloud-hosted guide more smoothly, add a login link at the index.xhtml

### CODE REVIEW

File: finish/frontend/src/main/webapp/index.xhtml [hunk 1]
```diff
@@ -14,6 +14,7 @@
 <body>
     <h1>Welcome to your Open Liberty application</h1>
     <p>Open Liberty is a lightweight open framework for building fast and efficient cloud-native Java microservices. Find out more at <a href="http://openliberty.io/" target="_blank" rel="noopener noreferrer">openliberty.io</a>.</p>
+    <p>Click the <a href="/login.jsf" target="_blank" rel="noopener noreferrer">login</a> to try out the application.</p>
```
Comment (dmuelle) [inline finish/frontend/src/main/webapp/index.xhtml:17]: Edit suggestion:

<p><a href="/login.jsf" target="_blank" rel="noopener noreferrer">Log in</a> to try out the application.</p>

File: start/frontend/src/main/webapp/index.xhtml [hunk 2]
```diff
@@ -14,6 +14,7 @@
 <body>
     <h1>Welcome to your Open Liberty application</h1>
     <p>Open Liberty is a lightweight open framework for building fast and efficient cloud-native Java microservices. Find out more at <a href="http://openliberty.io/" target="_blank" rel="noopener noreferrer">openliberty.io</a>.</p>
+    <p>Click the <a href="/login.jsf" target="_blank" rel="noopener noreferrer">login</a> to try out the application.</p>
```
Comment (dmuelle) [inline start/frontend/src/main/webapp/index.xhtml:17]: see previous comment

File: README.adoc [hunk 3]
```diff
@@ -418,11 +415,8 @@ endif::[]
 ifdef::cloud-hosted[]
 To launch the front-end web application, 
 select **Launch Application** from the menu of the IDE, type in **9090** to specify the port number for the front-end web application, 
-and click the **OK** button. You’re redirected to the **`https://accountname-9090.theiadocker-4.proxy.cognitiveclass.ai`** URL, 
-where **accountname** is your account name.
-
-In your browser, go to the front-end web application endpoint at the
-**`https://accountname-9090.theiadocker-4.proxy.cognitiveclass.ai/login.jsf`** URL. 
+and click the **OK** button. You’re redirected to the https://accountname-9090.theiadocker-4.proxy.cognitiveclass.ai URL, 
+where **accountname** is your account name. Click the **login** link at the welcome page.
```
Comment (dmuelle) [inline README.adoc:419]: Edit suggestion:
Click the **Log in** link on the welcome page.

**Review by dmuelle — CHANGES_REQUESTED** [review]
looks good, see my comments inline

File: finish/frontend/src/main/webapp/index.xhtml (hunk 1 above)
Comment (gkwan-ibm) [reply finish/frontend/src/main/webapp/index.xhtml]: updated

File: README.adoc (hunk 3 above)
Comment (gkwan-ibm) [reply README.adoc]: Updated

**Review by dmuelle — APPROVED** [review]
LGTM

**Pull request merged**